examples/config.yml

```diff
@@ -42,6 +42,13 @@ lighthouse:
   hosts:
     - "192.168.100.1"
 
+  # Toggle sending test packets to lighthouse hosts to assert the tunnels are still viable.
+  # Disabling sending test packets to lighthouses can reduce bookkeeping traffic
+  # and load on lighthouses with many tunnels.
+  # Setting to false may lead to connectivity issues if a lighthouse host is behind a NAT or firewall that
+  # tracks outbound packets from the lighthouse to the clients.
+  #test: true
```

Comment:
is `lighthouse.test` a specific enough key? 🤷 w/o this comment i probably wouldn't know what it's testing

Reply:
`lighthouse.send_test_packets = true`?